docs/site/Model.md

```diff
@@ -230,26 +216,48 @@ now:
   </thead>
 
   <tbody>
-
   <tr>
-  <td><code>description</code></td>
+    <td><code>name</code></td>
+    <td>String</td>
+    <td>None</td>
+    <td>Name of the model.</td>
+  </tr>
+  <tr>
+  <td><code>settings.description</code></td>
   <td>String</td>
   <td>None</td>
   <td>
     Optional description of the model. We only support string type for now. (see <a href="https://github.com/strongloop/loopback-next/issues/3428">issue #3428</a> for more discussion.)
   </td></tr>
 
   <tr>
-  <td><code>forceId</code></td>
-  <td>Boolean</td>
-  <td><code>true</code></td>
-  <td>
-    If true, prevents clients from setting the auto-generated ID value manually.
+    <td><code>settings.forceId</code></td>
+    <td>Boolean</td>
+    <td><code>true</code></td>
+    <td>
+      Set it to <code>true</code> to prevent clients from setting the auto-generated ID value manually.
+    </td>
+  </tr>
+
+  <tr>
+    <td><code>settings.hiddenProperties</code></td>
+    <td>Array of String</td>
+    <td><code>None</code></td>
+    <td>
+      The properties can be hidden from response bodies
+      (<code>.toJSON()</code> output). See <a href="#hidden-properties">Hidden properties</a> section below for details.
   </td>
   </tr>
 
   <tr>
-    <td><code>strict</code></td>
+    <td><code>settings.scope</code></td>
+    <td>Object</td>
+    <td>N/A</td>
+    <td>Scope enable you to specify commonly-used queries that you can reference as method calls on a model. See <a href="#scope">Scope</a> below for more details and examples.</td>
```

Comment:
`enable` -> `enables`
Just wondering, what does "method calls" mean?

Reply:
I just copied it from LB3 files :p I guess it's not the case if https://github.com/strongloop/loopback-next/pull/4829#discussion_r389977511  is not available in LB4.


How about: `Scope enables you to set a scope that will be applying to every query made by the repository of this model`?

And I will update examples as:

If you wish for a scope to be applied across all queries to **the repository of the model**, you can set, for example:

```
@model({
  settings: {
    scope: {
      'limit': 2,
    },
    idInjection: true
  }
})
export class Product extends Entity {
```

Now, any CRUD operation with a query parameter runs in the default scope will be applied; for example, assuming the above scope, a find operation such as

```ts
await ProductRepository.find({offset: 0});
```

Becomes the equivalent of this:

```ts
await ProductRepository.find({order: "name", offset: 0, limit: 100, where: {deleted: false}})
```

Reply:
Looks good to me! I would just reword a bit: `Scope enables you to set a scope that will apply to every query made by the model's repository`